packages/medusa/src/services/__tests__/event-bus.js

```diff
@@ -24,11 +24,18 @@ describe("EventBusService", () => {
         find: () => Promise.resolve([]),
       })
 
-      eventBus = new EventBusService({
-        manager: MockManager,
-        stagedJobRepository,
-        logger: loggerMock,
-      })
+      eventBus = new EventBusService(
+        {
+          manager: MockManager,
+          stagedJobRepository,
+          logger: loggerMock,
+        },
```

Comment:
suggestion: Since we are using that in multiple places I usually suggest to create a fake container up there and use it where it is needed